storage_devices/disk.py

```diff
@@ -311,4 +313,26 @@ def get_sysfs_addr(cls, device_id):
 
         return sysfs_addr.full_path
 
+    @classmethod
+    def identify(cls, device_name: str) -> bool:
+        output = TestRun.executor.run(
+            f"realpath /sys/block/{device_name}/device/driver | grep " f"virtio"
+        )
+        return output.exit_code == 0
+
 
+class DiskRegistry:
```

Comment:
maybe just hidden Disk class method __resolve_type? Do we need a separate class to do this?

Reply:
@mmichal10

Reply:
If you really want to have a separate class for one operation then at least rename it - a registry is sth that keeps records.

Reply:
Adding a method in `Disk` would lead to circular dependencies issue. 

Regarding the name of the class - it keeps info about all available types of disks so how about `DiskTypeRegistry`?

Reply:
Ah, yes - circles, got it :)  
DiskType is the enum with hdd..optane, so I would prefer that the class name would not resemble it. Maybe let's use "drive".  
Still not sure about the "registry" - there is a simple list inside (hardly a registry) and the only job this class does is finding the correct drive type. DriveTypeResolver? DriveTypeHelper? DriveTypes? Identifier3000?